docs/engineering/start-developing/bobaeditor.md

```diff
@@ -21,7 +23,9 @@ cd boba-editor
 yarn install
 ```
 
-## Development Instructions
+(If you would prefer to use a GUI with GitHub, you can [fork and clone](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop) via GitHub Desktop, then enter your new directory in a terminal and run `yarn install`.)
```

Comment:
You should use a "tip" callout for this.

:::tip
your text
:::

Reply:
Done!